src/samples/doxygen/first.cpp

```diff
@@ -0,0 +1,31 @@
+#include <rpp/rpp.hpp>
+#include <iostream>
+
+/**
+ * \example first.cpp
+ **/
+int main()
+{
+    //! [first]
+    rpp::source::just(1, 2, 3, 4, 5)
+        .first()
+        .subscribe(
+            [](const auto &v) { std::cout << "-" << v; },
+            [](const std::exception_ptr &error) {},
+            []() { std::cout << "-|" << std::endl; });
+    // Source: -1-2-3-4-5--|
+    // Output: -1-|
+    //! [first]
+
+    //! [first][empty]
```

Comment:
Doxygen foramt to detect samples looks like `! [SOMETHING MEANINFUL]`. Range before  two such a blocks uses as example. So, change it there and below
```suggestion
    //! [first_empty]
```